packages/material-ui/src/Badge/Badge.js

```diff
@@ -118,16 +143,31 @@ Badge.propTypes = {
    * Either a string to use a DOM element or a component.
    */
   component: componentPropType,
+  /**
+   * Show dot instead.
+   */
+  dot: PropTypes.bool,
```

Comment:
Possibly not, but I'm wondering if that should perhaps be a variant prop? Is there a possibility of other variants in the futire? I can't think of any, so just throwing it out there...